tests/tt_eager/python_api_testing/unit_testing/misc/test_softmax.py

```diff
@@ -162,67 +163,52 @@ def test_scale_mask_softmax_inplace(device, in_dtype, in0_mem_config, causal_mas
         attention_mask[mask] = float("-inf")
-        attention_mask32 = tilize_to_list(pad_weight(attention_mask))
-        attention_mask_t = ttl.tensor.Tensor(
-            attention_mask32,
-            [batch, 1, 32, seq_len],
-            # ttl.tensor.DataType.BFLOAT16,
-            ttl.tensor.DataType.FLOAT32 if in_dtype == ttl.tensor.DataType.FLOAT32 else ttl.tensor.DataType.BFLOAT16,
-            ttl.tensor.Layout.TILE,
-            device,
-            ttl.tensor.MemoryConfig(ttl.tensor.TensorMemoryLayout.INTERLEAVED, ttl.tensor.BufferType.L1),
-        )
+        attention_mask_t = ttnn.from_torch(attention_mask, dtype=mask_dtype, layout=ttnn.TILE_LAYOUT, device=device)
     else:
-        # attention_mask = torch.zeros(batch, 1, seq_len, seq_len)
         attention_mask = torch.rand(batch, 1, seq_len, seq_len)
         mask = torch.rand_like(attention_mask) < 0.2
         attention_mask[mask] = float("-inf")
-        attention_mask32 = tilize_to_list(pad_weight(attention_mask))
-        attention_mask_t = ttl.tensor.Tensor(
-            attention_mask32,
-            [batch, 1, seq_len, seq_len],
-            # ttl.tensor.DataType.BFLOAT16,
-            ttl.tensor.DataType.FLOAT32 if in_dtype == ttl.tensor.DataType.FLOAT32 else ttl.tensor.DataType.BFLOAT16,
-            ttl.tensor.Layout.TILE,
-            device,
-            ttl.tensor.MemoryConfig(ttl.tensor.TensorMemoryLayout.INTERLEAVED, ttl.tensor.BufferType.L1),
-        )
+        attention_mask = pad_weight(attention_mask)
+        attention_mask_t = ttnn.from_torch(attention_mask, dtype=mask_dtype, layout=ttnn.TILE_LAYOUT, device=device)
 
     input_tensor = torch.randn(input_shape).bfloat16().float()
-    in1_t = torch2tt_tensor(input_tensor, device, tt_memory_config=in0_mem_config, tt_dtype=in_dtype)
+    in1_t = ttnn.from_torch(
+        input_tensor, dtype=in_dtype, layout=ttnn.TILE_LAYOUT, device=device, memory_config=in0_mem_config
+    )
 
     if not is_grayskull():
-        if in_dtype == ttl.tensor.DataType.FLOAT32:
-            compute_kernel_config = ttl.tensor.WormholeComputeKernelConfig(
-                math_fidelity=ttl.tensor.MathFidelity.HiFi4,
+        if in_dtype == ttnn.float32:
+            compute_kernel_config = ttnn.WormholeComputeKernelConfig(
+                math_fidelity=ttnn.MathFidelity.HiFi4,
                 math_approx_mode=False,
                 fp32_dest_acc_en=True,
             )
         else:
-            compute_kernel_config = ttl.tensor.WormholeComputeKernelConfig(
-                math_fidelity=ttl.tensor.MathFidelity.HiFi4,
+            compute_kernel_config = ttnn.WormholeComputeKernelConfig(
+                math_fidelity=ttnn.MathFidelity.HiFi4,
                 math_approx_mode=False,
                 fp32_dest_acc_en=False,
             )
 
-    tt_output = ttl.operations.primary.transformers.scale_mask_softmax_in_place(
+    tt_output = ttnn.scale_mask_softmax_in_place(
```

Comment:
Same comment on scale_mask_softmax_in_place vs scale_mask_softmax

And I wonder if scale_mask could just be an optional kw_only argument, so we have a single api `softmax`

Reply:
yeah that's why I asked yesterday, should we cleanup the API now, or should we wait until it is moved completely. 
What do you think?